sdk/tests/deqp/functional/gles3/es3fNegativeBufferApiTests.js

```diff
@@ -481,12 +481,13 @@ goog.scope(function() {
         testGroup.addChild(new es3fApiCase.ApiCaseCallback(
             'copy_buffer_sub_data', 'Invalid gl.copyBufferSubData() usage', gl,
             function() {
-                var data = new Float32Array(32 * 32);
+                var data = new Float32Array(32);
```

Comment:
Agree. This is the only change that should be made. All of the changes below deviate from the original C++ test and should be removed.

Alternatively, you can just remove "data" and pass the number 32 explicitly to the gl.bufferData calls. The original test initializes the buffer objects to 32 bytes in size and that's what this test should do. WebGL guarantees that the buffer's contents will be zeroed out if you just pass the size.